### DISCUSSION TITLE
Prior on the slope parameter of the Hill function

### DISCUSSION THREAD

**Discussion by diayyash** [google/meridian#201] [Q&A]
Hello,

We're trying to apply a prior distribution on the slope parameter of the hill function, but whenever we change its default value ( Deterministic(1) ) to any other distribution, the model stops converging. Is there an explanation for this?

Thanks in advance.

**Comment by eddieligo** [comment]
The default value of `Deterministic(1.0)` for `slope_m` ensures a concave Hill function. Changing this prior to other distributions risks creating a not strictly concave Hill function, which can hinder the model's ability to distinguish the effects of different media channels. This can lead to poor model identifiability and difficulties for the MCMC sampler to explore the parameter space effectively, resulting in slow convergence. For more information about Hill function and identifiability issues, please see Section 2.2 in [Bayesian Methods for Media Mix Modeling with Carryover and Shape Effects](https://research.google/pubs/bayesian-methods-for-media-mix-modeling-with-carryover-and-shape-effects/).

Additionally, our current budget optimization algorithm relies on concave Hill curves to produce a global optimum; changing this prior may compromise this.

For now, we recommend using the default `Deterministic(1.0)` prior. Other priors can be considered, but they could make MCMC convergence more challenging.
